src/ODESolverTypes.f90

```diff
@@ -586,7 +597,8 @@ SUBROUTINE step_ODESolverType_Native(solver,t0,y0,tf,yf)
 !> methodology that was implmeneted which introduces substeps for lower order methods to generate historical
 !> data needed by the higher order methods.
 !>
-SUBROUTINE solve_bdf(f,myLS,ord,nstep,t,dt,yf,ydot,tol,ist,bdf_hist,updateJ_in)
+SUBROUTINE solve_bdf(f,myLS,ord,nstep,t,dt,yf,ydot,tol,ist,bdf_hist,updateJ_in,keepSolving_in)
```

Comment:
Need to have a `!> @param keepsolving_in` line up above for this new parameter

Reply:
done